packages/features/timezone-buddy/components/AvailabilitySliderTable.tsx

```diff
@@ -0,0 +1,165 @@
+import type { ColumnDef } from "@tanstack/react-table";
+import { ChevronLeftIcon, ChevronRightIcon } from "lucide-react";
+import { useMemo, useRef, useCallback, useEffect, useState } from "react";
+
+import dayjs from "@calcom/dayjs";
+import type { DateRange } from "@calcom/lib/date-ranges";
+import type { MembershipRole } from "@calcom/prisma/enums";
+import { trpc } from "@calcom/trpc";
+import { Avatar, Button, ButtonGroup, DataTable } from "@calcom/ui";
+
+import { TBContext, createTimezoneBuddyStore } from "../store";
+import { TimeDial } from "./TimeDial";
+
+export interface SliderUser {
+  id: number;
+  username: string | null;
+  email: string;
+  timeZone: string;
+  role: MembershipRole;
+  dateRanges: DateRange[];
+}
+
+export function AvailabilitySliderTable() {
```

Comment:
<img width="660" alt="Screenshot 2023-08-16 at 4 36 27 PM" src="https://github.com/calcom/cal.com/assets/53316345/35eeee37-0e69-47e4-b97b-fbed7a146510">

I had my availability set up between 9 - 4:30PM but i see 9 - 5PM

Reply:
Wait this is right isnt it? Yes youre only available for 1/2 of 4:30 but your still available in the 4 slot? I wonder if we should display an amber cell here let me try

Reply:
Screenshot above appears to be right for 9-4:30? 

Granted 4 shell should probably be amber. Will play around in figma to see if we can get something to show this.

Will tackle in a follow up if you believe this isnt a blocker @Udit-takkar

Reply:
@sean-brydon this is a bit confusing because 9 - 5 also shows this same UI. the screenshot shows slots 9 - 10, 10- 11, 11-12, 12 - 13, 13- 14, 14-15, 15- 16, 16-17.  the last slots should be 16:00 -  16:30. maybe because there isn't any clear boundary that help differentiate where does 4pm starts and ends. I expected 16:30 to be where "16" is displayed in the screenshot because 9:00 AM in the middle of 8 and 9 so middle of 16 and 17 would be 17:00 and not 16:30

Reply:
I understand - we need to make some compromises here. As we can't seperate into 30 min cells as we'd just get the same issue with 45 min offset timezones. @Jaibles has cleared this up in the designs for the selection hover
<img width="225" alt="CleanShot 2023-08-17 at 09 19 10@2x" src="https://github.com/calcom/cal.com/assets/55134778/6e21a012-4e4f-4c4b-a470-29b183f399b0">